### ISSUE TITLE
Remove Cocoapods and Carthage support

### ISSUE THREAD

**Issue by alanzeino** [uber/ios-snapshot-test-case#160]
Just a heads up that we are considering removing support for these considering recent issues running these tools on Apple Silicon, Apple's changes to xcframeworks, Swift Package Manager gaining steam in the past year, etc.

If you use one of these integration points consider at least switching to SwiftPM integration. We may drop Cocoapods integration early next year considering the above.

**Comment by pwadowski** [comment]
Do you still have plans to drop support for Cocoapods? I am researching options to migrate PointFree snaphot testing lib in my client's project to something which supports Cocoapods and your lib is the only solution which I found.

**Comment by alanzeino** [comment]
Yes likely this year.

Even if we dropped Cocoapods you could just copy the code into your app and use the `:path`, or use an old version of the library.

**Comment by pwadowski** [comment]
Yeah, I know but we have a flow which automatically updates our pods and this solution won't fit into this. Probably, we are going to try use both cocoapods and spm in one project.
Thanks for quick response.
